copier/copier.go

```diff
@@ -343,22 +343,24 @@ func Stat(root string, directory string, options StatOptions, globs []string) ([
 
 // GetOptions controls parts of Get()'s behavior.
 type GetOptions struct {
-	UIDMap, GIDMap     []idtools.IDMap   // map from hostIDs to containerIDs in the output archive
-	Excludes           []string          // contents to pretend don't exist, using the OS-specific path separator
-	ExpandArchives     bool              // extract the contents of named items that are archives
-	ChownDirs          *idtools.IDPair   // set ownership on directories. no effect on archives being extracted
-	ChmodDirs          *os.FileMode      // set permissions on directories. no effect on archives being extracted
-	ChownFiles         *idtools.IDPair   // set ownership of files. no effect on archives being extracted
-	ChmodFiles         *os.FileMode      // set permissions on files. no effect on archives being extracted
-	StripSetuidBit     bool              // strip the setuid bit off of items being copied. no effect on archives being extracted
-	StripSetgidBit     bool              // strip the setgid bit off of items being copied. no effect on archives being extracted
-	StripStickyBit     bool              // strip the sticky bit off of items being copied. no effect on archives being extracted
-	StripXattrs        bool              // don't record extended attributes of items being copied. no effect on archives being extracted
-	KeepDirectoryNames bool              // don't strip the top directory's basename from the paths of items in subdirectories
-	Rename             map[string]string // rename items with the specified names, or under the specified names
-	NoDerefSymlinks    bool              // don't follow symlinks when globs match them
-	IgnoreUnreadable   bool              // ignore errors reading items, instead of returning an error
-	NoCrossDevice      bool              // if a subdirectory is a mountpoint with a different device number, include it but skip its contents
+	UIDMap, GIDMap        []idtools.IDMap   // map from hostIDs to containerIDs in the output archive
+	Excludes              []string          // contents to pretend don't exist, using the OS-specific path separator
+	ExpandArchives        bool              // extract the contents of named items that are archives
+	ChownDirs             *idtools.IDPair   // set ownership on directories. no effect on archives being extracted
+	ChmodDirs             *os.FileMode      // set permissions on directories. no effect on archives being extracted
+	ChownFiles            *idtools.IDPair   // set ownership of files. no effect on archives being extracted
+	ChmodFiles            *os.FileMode      // set permissions on files. no effect on archives being extracted
+	Parents               bool              // maintain the sources parent directory in the destination
```

Comment:
Are the parent directories of the items being read supposed to be emitted in the tarstream when this flag is set?

Reply:
I have made a rework and the current version copies the parent directories to the tarstream.